### ISSUE TITLE
Warning about `this`.

### ISSUE THREAD

**Issue by antony** [6eDesign/svelte-calendar#29]
In ES modules, `this` function binding is invalid. There is no longer a need to bind to `this` after conversion from Svelte 2.

```js
src/components/related-adventures/RelatedAdventures.svelte changed. rebuilding...
• server
The 'this' keyword is equivalent to 'undefined' at the top level of an ES module, and has been rewritten
45: 
46:   onMount(() => {
47:     document.addEventListener('click',checkForFocusLoss.bind(this)); 
                                                                 ^
48:     if(!trigger) return; 
49:     triggerContainer.appendChild(trigger.parentNode.removeChild(trigger));
```

I think the fix for this is simply to change to `checkForFocusLoss`, with no binding.

Happy to fix this and open a PR if you agree.

**Comment by 6eDesign** [comment]
Hey @antony - this has been removed and will be published shortly (v1.0.5).

**Comment by antony** [comment]
@6eDesign stellar! :D
